sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -47,45 +47,28 @@ The authorization flow for resource server:
+| **azure.activedirectory**.allow-telemetry                        | If telemetry events should be published to Azure AD. The default value is true. |
+| **azure.activedirectory**.app-id-uri                             | Used in the "aud" claim of an id token.                      |
+| **azure.activedirectory**.authorization-clients                  | Resource server name that the application is going to visit. |
+| **azure.activedirectory**.authorization-clients.{client-name}.scopes | API permissions of a resource server that the application is going to acquire. |
+| **azure.activedirectory**.authorization-clients.{client-name}.scopes.on-demand | If the authorization flow of current resource server will be executed on demand. The default value is false. |
+| **azure.activedirectory**.base-uri                               | Base uri for authorization server, the default value is `https://login.microsoftonline.com/`. |
+| **azure.activedirectory**.client-id                              | Registered application ID in Azure AD.                       |
+| **azure.activedirectory**.client-secret                          | API Access Key of the registered application.                |
+| **azure.activedirectory**.graph-membership-uri                   | The way this starter uses to load users' membership depends on the configured membership URI. By default, the starter uses `https://graph.microsoft.com/v1.0/me/memberOf` to get direct membership of current user. <br>To get all transitive membership, you can configure it as `https://graph.microsoft.com/v1.0/me/transitiveMemberOf`. |
+| **azure.activedirectory**.jwk-set-cache-refresh-time             | The refresh time of the cached JWK set before it expires, default is 5 minutes. |
+| **azure.activedirectory**.jwt-connect-timeout                    | Connection Timeout for the JWKSet Remote URL call.           |
+| **azure.activedirectory**.jwt-read-timeout                       | Read Timeout for the JWKSet Remote URL call.                 |
+| **azure.activedirectory**.jwt-set-cache-lifespan                 | The lifespan of the cached JWK set before it expires, default is 5 minutes. |
+| **azure.activedirectory**.jwt-size-limit                         | Size limit in Bytes of the JWKSet Remote URL call.           |
+| **azure.activedirectory**.post-logout-redirect-uri               | Redirect uri for posting log-out.                            |
+| **azure.activedirectory**.redirect-uri-template                  | Used by the authorization server to return responses containing authorization credentials to the client via the resource owner user-agent. |
+| **azure.activedirectory**.session-stateless                      | If true, the stateless auth filter `AADAppRoleStatelessAuthenticationFilter` will be activated. By default, is value is false, which activates `AADAuthenticationFilter`. |
```

Comment:
Should we add `Deprecated` here? Because the 2 filter is deprecated.

Reply:
How about now?